dotCMS/src/integration-test/java/com/dotcms/security/apps/AppsAPIImplTest.java

```diff
@@ -1405,4 +1408,37 @@ public void Test_NPE_Free_ToString(){
         assertNotNull(formNotAllNull.toString());
     }
 
+    @Test
```

Comment:
Approved. Just add a comment to describe this test please.

Reply:
done